pkg/forwarder/internal/retry/point_count_telemetry.go

```diff
@@ -14,24 +14,25 @@ import (
 	"go.uber.org/atomic"
 )
 
-// PointDroppedSender sends the number of points dropped
-type PointDroppedSender struct {
+// PointCountTelemetry sends the number of points successfully sent and the number of points dropped.
```

Comment:
Shouldn't all these "Point" and "point" be plural? `PointsSuccessFullysent`, `pointsCountTelemetry`, even the metrics `datadog.agent.points.dropped` and `datadog.agent.points.sent`, etc.

Reply:
I used the same naming as https://github.com/DataDog/datadog-agent/blob/7.40.0/pkg/aggregator/aggregator.go#L594,